reporter-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "reporterEnabled": "spec, json",
```

Comment:
is there not a dot missing here? Just wondering.

```suggestion
  "reporterEnabled": ".spec, .json",
```

Reply:
No those are names of reporters not file extensions